test/end-to-end/specs/viewBlueprint.test.js

```diff
@@ -14,6 +14,7 @@ describe("View Blueprint Page", function() {
 
   before(function() {
     commands.login();
+    commands.startLoraxIfItDoesNotStart();
```

Comment:
You have added this extra step to enable and start lorax-composer in each test.
If that single component will break, all tests will break in this way.
For this reason it would make more sense to have it by default enabled, and don't enable/start it from the UI for all the tests, apart from the single test that it's checking this feature. And for that one you could stop it with 'systemctl stop' in the setUp of the test.

Reply:
@KKoukiou appreciate your suggestion! :+1: 
The "start lorax-composer" is only got run with the first test, in this case it's api.test.js. The rest of tests will not run this extra step. If I add this test, i'll add it in api.test.js. So It's almost the same thing for the test. The only reason I choose this solution is saving one more step - stop lorax-composer.